src/_includes/subnav.html

```diff
@@ -0,0 +1,16 @@
+{% if page.next or page.prev -%}
+<nav id="subnav">
```

Comment:
this will output two navs with the same ID which is not good practice, we may want to change this to a class?

Reply:
Yes, thanks! Done